### DISCUSSION TITLE
【体验报告】快来分享您的部署经验，加入TCA大家庭！

### DISCUSSION THREAD

**Discussion by cocorunning** [Tencent/CodeAnalysis#229] [Show and tell]
镇楼帖~~
欢迎大家踊跃畅聊部署经验！可以是部署心得、功能体验、成果分享、吐槽（要合理哎喂...）~
总之，尽情来吧！

**Comment by shang-chengxiang** [comment]
欢迎交流TCA在腾讯云Lighthouse部署经验
腾讯云+社区文章：https://cloud.tencent.com/developer/article/1936266
推荐各位选择海外地域的可用区、2C4G以上的配置安装部署
Docker的部署难度相对更加简单，故推荐大家使用Docker部署

**Comment by shang-chengxiang** [comment]
腾讯云Lighthouse是在CVM的基础上简化的一款计算类产品
CVM有更丰富的功能，但是LIghthouse也是足够日常使用
GPU机型暂时没有相关的使用经验，后期会进行相关补充
